azurerm/helpers/authentication/config.go

```diff
@@ -9,10 +9,12 @@ import (
 	"github.com/Azure/go-autorest/autorest/azure/cli"
 )
 
+// TODO: separate objects for Input/Output
```

Comment:
Should there be a GitHub issue created for these TODOs?

Reply:
they'll be coming in the next PR - so it should be fine